### PR TITLE
[cdc]add jdbc properties configuration

### PR DESCRIPTION
**Pull request by vinlee19**
# Proposed changes
In the previous version, we were unable to configure the JDBC URL properties for Postgres, SqlServer, and Db2, which might cause some connection issues in certain scenarios.  
Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/db2/Db2DatabaseSync.java [hunk 1]
```diff
@@ -89,12 +89,26 @@ public void registerDriver() throws SQLException {
 
     @Override
     public Connection getConnection() throws SQLException {
+        Properties jdbcProperties = getJdbcProperties();
+        StringBuilder jdbcParams = new StringBuilder();
+        int cnt = 0;
```
Comment (DongLiang-0) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/db2/Db2DatabaseSync.java:94]: Is it more appropriate to use boolean here?
```suggestion
       boolean firstParam = true;
```

**Comment by JNSimba** [comment]
Add the description of PR, thanks.

**Comment by vinlee19** [comment]
> Add the description of PR, thanks.

done

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**